recipes/pygridtools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+    name: pygridtools
+    version: 0.2
+
+source:
+    git_url: https://github.com/Geosyntec/pygridtools.git
+    git_tag: v0.2
```

Comment:
Not sure if that is necessary when using git. Git is pretty robust to ensure that we have the "correct" clone. (We must add checksum if using files though.)